<launch>
  <arg name="base" value="base_link"/>
  <arg name="map" value="odom"/>
  <!-- odom map odom_combined -->
 <arg name="cmd_vel" value="cmd_vel"/>

<include file="$(find realsense2_camera)/launch/rs_t265.launch">

</include>
    <node pkg="create" type="create_tfControl_node" name="slave1" output="screen" > 
      <!-- PID参数调节  -->
      <param name="Scalex_P" type="double" value="5" />
      <param name="Scalex_I" type="double" value="0.3" />
      <param name="Scalez_P" type="double" value="5" />
      <param name="Scalez_I" type="double" value="0.15" />
      <param name="max_vel" type="double" value="1" />
      <param name="normal_vel" type="double" value="0.5" />
      <param name="roworleng" type="double" value="1" /> 
      <param name="plusandminus" type="double" value="1.3" />
      
      <param name="base_to_car" type="string" value="$(arg base)" />
      <param name="base_frame" type="string" value="$(arg map)" /> 
      <remap from="cmd_vel_ori" to="cmd_vel" />
    </node>

 
</launch>